SA/design_concept/content/tool_research/enso.typ

```diff
@@ -0,0 +1,16 @@
+= Enso
+Enso is a functional programming language created by Enso International Inc., designed for data science.
+Programs can be created by both text and a visual editor.
+The visual editor allows a user to define components that can be connected with each other, symbolizing the data-flow from one component to another.
```

Comment:
Maybe add an empty line in here somewhere to create 2 paragraphs and break the block.